src/spinner/spinner.style.ts

```diff
@@ -0,0 +1,32 @@
+import { css } from "lit";
+
+export default css`
+  :host {
+    box-sizing: border-box;
+  }
+
+  .spinner {
+    width: 20px;
+    height: 20px;
+    padding: 2px;
+    stroke-linecap: round;
+  }
+
+  .rotating {
+    transform-origin: 300px 300px;
+    animation : rotate 1s linear infinite;
+  }
+
+  .primary {
+    color: var(--tap-sys-color-surface-black);
```

Comment:
You can add a token for this field and use `--tap-sys-color-surface-black` as a default value:

```suggestion
    color: var(--tap-spinner-color-primary, var(--tap-sys-color-surface-black));
```

With this approach, the user of the component can override the color of the spinner.